src/Analyzer.Utilities.UnitTests/JsonPathResolverTests.cs

```diff
@@ -255,6 +235,174 @@ public void ResolveResourceType_JObjectWithResourcesNotArrayOfObjects_ReturnsEmp
             Assert.AreEqual(0, new JsonPathResolver(jToken, jToken.Path).ResolveResourceType("anything").Count());
         }
 
+        [TestMethod]
+        public void ResolveResourceType_LanguageVersion2SymbolicNaming_ReturnsResourcesOfCorrectType()
```

Comment:
To be very explicit, can you update a test (if one exists)/add a new test for `ResolveResourceType_LanguageVersion1SymbolicNaming_ReturnsResourcesOfCorrectType()`? It could help catch errors if someone makes updates to see if it was a lang version error